ci/pipelines/resources/template.jsonnet

```diff
@@ -36,7 +36,6 @@ local build_params =
   else if resource == "concourse-pipeline" then
     {
       build: ".",
-      dockerfile: resource+"-resource/Dockerfile"
```

Comment:
Don't need this; I ended up closing https://github.com/concourse/concourse-pipeline-resource/pull/52. Either way this value will still need to be set so it knows where the `Dockerfile` is.

Reply:
The `dockerfile` value would be taken from the default place which is at https://github.com/concourse/concourse/blob/dbaac83eea403e425f71ac6387303f5117da2296/ci/pipelines/resources/template.jsonnet#L247-L252

Reply:
ahh didn't notice that :+1: